StoreApp/Controllers/AccountController.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Web.Mvc;
+using AutoMapper;
+using StoreApp.BLL.DTO;
+using StoreApp.BLL.Services;
+using StoreApp.Models.Account;
+
+namespace StoreApp.Controllers
+{
+    public class AccountController : Controller
+    {
+        private UserValidateService userValidateService;
+        private UserService userService;
+        private IMapper config;
+        public AccountController()
+        {
+            userValidateService = new UserValidateService();
+            userService = new UserService();
+            config = new MapperConfiguration(cfg => cfg.CreateMap<UserLoginViewModel, UserDTO>()).CreateMapper();
+        }
+        
+        [HttpGet]
+        public ActionResult Registration()
+        {
+            ViewBag.User = new UserRegistrationViewModel();
+            return View();
+        }
+
+        [HttpPost]
+        public ActionResult Registration(UserRegistrationViewModel model)
+        {
+            if (!ModelState.IsValid)
+            {
+                ViewBag.Message = "Your registraition wrong!";
```

Comment:
Look at the message, there is the grammar mistake)

Reply:
fixed